unittests/FEXLinuxTests/tests/signal/invalid_hlt.cpp

```diff
@@ -12,14 +12,13 @@
 extern "C" void IntInstruction();
 
 #pragma GCC diagnostic ignored "-Wattributes" // Suppress warning in case control-flow checks aren't enabled
-__attribute__((naked, nocf_check))
-  static void InvalidINT() {
+__attribute__((naked, nocf_check)) static void InvalidINT() {
```

Comment:
Should we enable `BreakAfterAttributes`?

CC @lioncash , since this seems analogous to breaking after `template<>`.

If we do this, we may want to add these macros to `AttributeMacros`: `JEMALLOC_NOTHROW`, `FEX_ALIGNED`, `FEX_ANNOTATE`, `FEX_DEFAULT_VISIBILITY`, `FEX_NAKED`, `FEX_PACKED`, `FEXCORE_PRESERVE_ALL_ATTR`, `GLIBC_ALIAS_FUNCTION`